squid/tests/test_squid.py

```diff
@@ -26,3 +27,34 @@ def test_check_ok(aggregator, check, instance):
     for metric in common.EXPECTED_METRICS:
         aggregator.assert_metric("squid.cachemgr." + metric, tags=expected_tags)
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.parametrize(
+    'raw_version, version_metadata, count',
+    [
+        (
+            'squid/3.0.3',
+            {'version.scheme': 'semver', 'version.major': '3', 'version.minor': '0', 'version.raw': '3.0.3'},
+            5,
+        ),
+        (
+            'squid/1.4.5',
+            {'version.scheme': 'semver', 'version.major': '1', 'version.minor': '4', 'version.raw': '1.4.5'},
```

Comment:
Unless I miss something you can check for `version.patch` here as well.

Reply:
Yes, I completely left that out! Added it!